tests/test_algorithm.py

```diff
@@ -545,16 +534,29 @@ def setUpClass(cls):
         cls.env.write_data(equities_data=equities_metadata,
                            futures_data=futures_metadata)
 
-        cls.data_portal = create_data_portal(
-            cls.env,
-            cls.tempdir,
-            cls.sim_params,
-            cls.sids
-        )
+        trades_by_sid = {}
+        for sid in cls.sids:
+            trades_by_sid[sid] = factory.create_trade_history(
```

Comment:
This is the same trade history as on master, correct? If so, :+1:. Having the same inputs/outputs (where possible) will reconciling the implementations.